nservicebus/licensing/index.md

```diff
@@ -37,3 +37,5 @@ partial: limitations
 There are several options available for installing the license file. 
 
 partial: license-management
+
+NOTE: The easiest way to globally manage a license is by importing the license via the ServiceControl Management Utility. All platform tools and NServiceBus will detect the license  as the license will be imported at the machine level.
```

Comment:
In the v6 version of the page, it appears there there is already a paragraph in the license-management partial that is attempting to address this point. Perhaps that text should be tweaked instead if needed?

This note also doesn't reflect the fact that you can be running on mac and linux, where the management utility doesn't exist.

Reply:
For v7 dotnetcore, we wont hit the registry by default either right?

Reply:
Correct. We only look at the registry locations when running on the .NET Framework.